src/bubble_swarm/angle_area.rs

```diff
@@ -0,0 +1,31 @@
+use crate::math2d::regioned_angle::{RegionedAngle, RegionedAngleTrait};
+
+#[derive(Debug, Clone, Copy, PartialEq, PartialOrd)]
+pub struct AngleArea {
+    area: f32,
+    nb_angles: usize,
+}
```

Comment:
For now this implementation is sufficient. I want to try out an offset_angle as a member soon. Because it is totally imaginable that you want an angle to be in the segment from thirty degrees to 90 degrees for example :)